plugins/modules/rds_cluster_info.py

```diff
@@ -0,0 +1,306 @@
+#!/usr/bin/python
+# Copyright (c) 2022 Ansible Project
+# Copyright (c) 2022 Alina Buzachis (@alinabuzachis)
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+DOCUMENTATION = r'''
+module: rds_cluster_info
+version_added: 3.1.0
```

Comment:
```suggestion
version_added: 3.2.0
```

if we made it ...but maybe we delay it when the CI/CD stays broken...?

Reply:
I added `version_added: 3.3.0 `for the moment.